app/helpers/mustache_helper.rb

```diff
@@ -64,8 +64,16 @@ def js_variables
   end
 
   def js_files
-    [{ path: styleguide_path('/js/dist/require.js'),
-       data_main: styleguide_path('/js/dist/main/main') }]
+    js_entry_point = ENV['JS_ENTRYPOINT'] || '/js/dist/'
+    js_entry_point = js_entry_point.dup << '/' unless js_entry_point.end_with?('/')
+
+    log = Logger.new(STDOUT)
+    log.level = Logger::WARN
+    log.warn("js_entry_point = " + js_entry_point)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.